packages/amplify-graphql-http-transformer/src/graphql-http-transformer.ts

```diff
@@ -288,11 +324,23 @@ function createResolver(stack: cdk.Stack, dataSourceId: string, context: Transfo
           ref('util.qr($util.appendError($ctx.result.body, $ctx.result.statusCode))'),
         ),
       ),
+      `${functionId}.res.vtl`,
+    ),
+    dataSourceId,
+    stack,
+  );
+
+  return context.api.host.addResolver(
+    config.resolverTypeName,
+    config.resolverFieldName,
+    MappingTemplate.inlineTemplateFromString(printBlock('Stash resolver specific context.')(compoundExpression(requestTemplate))),
+    MappingTemplate.s3MappingTemplateFromString(
+      '$util.toJson($ctx.prev.result)',
       `${config.resolverTypeName}.${config.resolverFieldName}.res.vtl`,
```

Comment:
nit: since this is the last appsync function can we use
```suggestion
      `${config.resolverTypeName}.${config.resolverFieldName}.after.res.vtl`,
```

Reply:
This one isn't an AppSync function - it's the resolver mapping template. I don't think we use `.after.res.vtl` anywhere else in the codebase.